### PR TITLE
set user language for request in backend

### PR DESCRIPTION
**Pull request by danrot**
__tasks:__

- [x] test coverage

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| Related PRs      | none
| BC breaks        | The language of the requests have the user language instead of the default one
| Documentation PR | none

### CODE REVIEW

**Comment by danrot** [comment]
@mob8607 Would be nice if you could test if that works for you.

**Comment by webmozart** [comment]
What was the bug that occurred before this fix?

**Comment by wachterjohannes** [comment]
@danrot tests failing

**Comment by danrot** [comment]
@webmozart There was no bug in Sulu itself, but in another project they were using the translator in a service. And the translator returned the wrong translations, because the locale was missing on the request.

File: src/Sulu/Bundle/SecurityBundle/EventListener/UserLocaleListener.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    {
+        $this->tokenStorage = $tokenStorage;
+    }
+
+    /**
+     * Sets the locale of the current User to the request, if a User is logged in.
+     *
+     * @param GetResponseEvent $event
+     */
+    public function onKernelRequest(GetResponseEvent $event)
```
Comment (webmozart) [inline src/Sulu/Bundle/SecurityBundle/EventListener/UserLocaleListener.php]: I'd use a more descriptive name here, like `copyUserLocaleToRequest()`.

Comment (danrot) [inline src/Sulu/Bundle/SecurityBundle/EventListener/UserLocaleListener.php]: done

**Pull request merged**